<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileCard</class>
 <widget class="QWidget" name="fileCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>169</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="cardLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QToolButton" name="btnOpen">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblFileName">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btnClose">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
